mods/ra/rules/vehicles.yaml

```diff
@@ -24,14 +24,23 @@ V2RL:
 		Range: 4c0
 	Armament:
 		Weapon: SCUD
+	AmmoPool:
+		Ammo: 1
+		PipCount: 0
+		AmmoCondition: ammo
+	ReloadAmmoPool:
+		Delay: 240
```

Comment:
It might be a good idea to reduce the weapon reload delay to 0 to avoid future bugs or confusion when the two get out of sync.

Reply:
But you will also need to add a `PauseOnCondition: !ammo` to prevent the weapon from firing when there is no ammo!